apps/examples/sensorbd_demo/examples/uart_loopback.c

```diff
@@ -69,7 +69,6 @@ void uartloopback_main(int argc, char *argv[])
 			if (write(fd, buf_tx, 5) < 0) {
 				ret = 1;
 			}
-
```

Comment:
Adding an empty line for better readability is a good thing. I don't understand why you want to remove this line.